static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -12,9 +12,10 @@ Once initialized in a project, DVC populates its installation directory
   The local config file can be edited by hand or with a special command:
   `dvc config --local`.
 
-- `.dvc/cache` - the cache directory will contain your data files. (The data
-  directories of DVC repositories will only contain links to the data files in
-  the cache, refer to
+- `.dvc/cache` - the
+  [cache directory](/docs/user-guide/dvc-files-and-directories#structure-of-cache-directory)
```

Comment:
try just `#structure-of-cache-directory` - should be enough I hope when you do a link within the same document